### PR TITLE
sdl_mixer: Enable MP3

### PR DESCRIPTION
**Pull request by lightbinder**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Some time ago, the SDL project pulled its old SDL1 source files offline.  They are no longer reachable in their original location.  I've corrected that in the formula.  I've also enabled MP3 support, as it was disabled in the original formula.

### CODE REVIEW

**Comment by Bo98** [comment]
> Some time ago, the SDL project pulled its old SDL1 source files offline. They are no longer reachable in their original location. I've corrected that in the formula.

Still there for me?

```
$ curl -s https://www.libsdl.org/projects/SDL_mixer/release/SDL_mixer-1.2.12.tar.gz | sha256sum
1644308279a975799049e4826af2cfc787cad2abb11aa14562e402521f86992a  -
```

**Comment by lightbinder** [comment]
Interesting.  Curl has been consistently refusing to pull it for at least months now on my end, but it looks like it just might be me.  My apologies!

-slp
________________________________

**Comment by Bo98** [comment]
I would be fine with adding SourceForge as a mirror (though you can trim that query string off the end of the URL - see other formulae which use SourceForge fine without it).

**Comment by lightbinder** [comment]
I'll work on that and submit again then!

-slp
________________________________

**Comment by lightbinder** [comment]
Did as @Bo98 asked and made SF a mirror.

**Comment by ywwry66** [comment]
```
curl -sL https://downloads.sourceforge.net/project/libsdl/SDL/1.2.12/SDL-1.2.12.tar.gz | sha256sum
df77eb883efdcc61220eaf7ade38324629c273c278ce8d700a5530369e9b6df9  -
```
Checksum is different

**Comment by lightbinder** [comment]
> ```
> curl -sL https://downloads.sourceforge.net/project/libsdl/SDL/1.2.12/SDL-1.2.12.tar.gz | sha256sum
> df77eb883efdcc61220eaf7ade38324629c273c278ce8d700a5530369e9b6df9  -
> ```
> 
> Checksum is different

Removed the mirror altogether -- this pull request only enables MP3 support now.

File: Formula/sdl_mixer.rb [hunk 1]
```diff
@@ -34,6 +34,8 @@ class SdlMixer < Formula
   depends_on "libogg"
   depends_on "libvorbis"
   depends_on "sdl12-compat"
+  depends_on "libmad"
```
Comment (ywwry66) [inline Formula/sdl_mixer.rb:37]:
```suggestion
  depends_on "mad"
  depends_on "sdl12-compat"
```

**Pull request closed without merging**